docs/src/modules/components/DemoPropsForm.tsx

```diff
@@ -419,7 +419,7 @@ export default function ChartDemoPropsForm<T extends { [k: string]: any } = {}>(
                       {resolvedValue}
                     </Box>
                     {/* void */}
-                    <Box />
+                    <div />
```

Comment:
Is their a reason?

Reply:
Yes, after the monorepo bump, we have a new rule, which does not allow using empty `Box` elements: https://github.com/mui/material-ui/pull/38461